src/nwbinspector/checks/behavior.py

```diff
@@ -7,7 +7,11 @@
 
 @register_check(importance=Importance.CRITICAL, neurodata_type=SpatialSeries)
 def check_spatial_series_dims(spatial_series: SpatialSeries):
-    """Check if a SpatialSeries has the correct dimensions."""
+    """
+    Check if a SpatialSeries has the correct dimensions.
+
+    Best Practice: :ref:`_best_practice_unit_of_measurement`
```

Comment:
There could be a specific section to discuss these particular checks, but for now just linking to the common one

Reply:
I'll write this section now